src/solvers/dg_tree/dg.jl

```diff
@@ -139,7 +139,7 @@ Create a discontinuous Galerkin method.
 If [`basis isa LobattoLegendreBasis`](@ref LobattoLegendreBasis),
 this creates a [`DGSEM`](@ref).
 """
-struct DG{RealT, Basis<:AbstractBasisSBP{RealT}, Mortar, SurfaceFlux, VolumeIntegral}
+struct DG{RealT, Basis, Mortar, SurfaceFlux, VolumeIntegral}
```

Comment:
```suggestion
struct DG{Basis, Mortar, SurfaceFlux, VolumeIntegral}
```
As discussed on Slack, you should also change https://github.com/trixi-framework/Trixi.jl/blob/c635c6afb1db8b6f6cefc72d1065a111bc730081/src/solvers/dg_tree/dg.jl#L179 to 
```julia
 @inline Base.real(dg::DG) = real(dg.basis) 
```